central/node/datastore/internal/store/dackbox/store_impl.go

```diff
@@ -313,27 +313,23 @@ func (b *storeImpl) writeNodeParts(parts *NodeParts, clusterKey []byte, iTime *p
 
 	var componentKeys [][]byte
 	// Update the node components and cves iff the node upsert has updated scan.
-	// Note: In such cases, the loops in following block will not be entered anyways since len(parts.children) and len(parts.nodeCVEEdges) is 0.
+	// Note: In such cases, the loops in following block will not be entered anyways since len(parts.Children) and len(parts.NodeCVEEdges) is 0.
```

Comment:
you can remove the comment about `len(parts.NodeCVEEdges)` since that's gone now